app/main.py

```diff
@@ -1,8 +1,30 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    person_list = [
+        Person(person_data["name"], person_data["age"])
+        for person_data in people
+    ]
+
+    for person in people:
+
+        if "wife" in person:
+            if person.get("wife"):
```

Comment:
.get does 2 things:
1. Checks if this key exist. If no -> returns default value(None) and KeyError will not be raised
2. If key exists -> returns its value, this you can simplify these if statements